lib/install/bin/webpack-dev-server.tt

```diff
@@ -1,27 +1,40 @@
 <%= shebang %>
+$stdout.sync = true
+
 require 'shellwords'
+require 'json'
 
 ENV['RAILS_ENV'] ||= 'development'
-RAILS_ENV   = ENV['RAILS_ENV']
+RAILS_ENV = ENV['RAILS_ENV']
 
 ENV['NODE_ENV'] ||= RAILS_ENV
-NODE_ENV    = ENV['NODE_ENV']
+NODE_ENV = ENV['NODE_ENV']
 
 APP_PATH = File.expand_path('../', __dir__)
 ESCAPED_APP_PATH = APP_PATH.shellescape
 
-SET_NODE_PATH  = "NODE_PATH=#{ESCAPED_APP_PATH}/node_modules"
-WEBPACKER_BIN  = "./node_modules/.bin/webpack-dev-server"
-WEBPACK_CONFIG = "#{ESCAPED_APP_PATH}/config/webpack/#{NODE_ENV}.js"
+begin
+  package_json = File.read(File.join(APP_PATH, 'package.json'))
+  config = JSON.parse(package_json)
 
-# Warn the user if the configuration is not set
-RAILS_ENV_CONFIG = File.join("config", "environments", "#{RAILS_ENV}.rb")
+  NODE_MODULES_PATH = "#{File.join(ESCAPED_APP_PATH, config['webpacker']['nodeModulesPath'])}"
+  WEBPACK_CONFIG_PATH = "#{File.join(ESCAPED_APP_PATH, config['webpacker']['configPath'])}"
+  PACKS_PATH = "#{File.join(ESCAPED_APP_PATH, config['webpacker']['distPath'])}"
+  WEBPACK_BIN  = "#{NODE_MODULES_PATH}/.bin/webpack-dev-server"
+  WEBPACK_CONFIG = "#{WEBPACK_CONFIG_PATH}/development.hot.js"
```

Comment:
Given all the known issues with hot reloading, maybe it shouldn't be the default.